Homeword submission/stylesheet.css

```diff
@@ -0,0 +1,140 @@
+.container{
+	margin: 0 auto;
+	width: 900px;
+}
+.text{
+	background: url(./header_bg.jpg);
+	background-size: cover;
+	height: 375px;
+	width: 95%;
+	display: inline-block;
+}
+h2{
+	margin-top: 40px;
+	margin-left: 15px;
+	font-style: extra bold;
+	color: white;
+	text-align: left;
+	text-indent: 50px;
+	font-size: 24px;
+}
+p{
+	font-style: semi bold;
+	color: white;
+	text-align: left;
+	text-indent: 200px;
+	font-size: 24px;
+	margin-top: 50px;
+}
+.button{
+    vertical-align: middle;
+	height: 100px;
+	text-align: center;
+	font-style: extra bold;
+	font-size: 14px;
+	border: .5px solid black;
+	border-radius: 5px;
+	line-height: 43px;
+	background-color: yellow;
+	width: 20%;
+	height: 40px;
+	margin: 40px auto;
+}
+.maintext{
+	background-color: #f0efef;
+	margin: 0;
+	padding: 0;
+	border: 0;
+	width: 95%;
+	display: inline-block;
+	height: 300px;
+	marpx;
+}
+.benefits{
+	font-size: 20px;
+	font-style: semibold;
+	color: #606161;
+	text-align: left;
+	text-indent: 50px;
+}
+.the{
+	font-style: light;
+	font-size: 14px;
+	color: #000000;
+	text-indent: 50px;
+	margin-top: 0%;
+}
+.list{
+	font-style: light;
+	font-size: 14px;
+	text-indent: 100px;
+	margin-top: 25px;
+}
+.bottom{
+	background: url(./testimonial_bg.jpg);
+	display: inline-block;
+	width: 95%;
+	height: 200px;
+}
+.test1{
+	font-style: light italic;
+	font-size: 18px;
+	text-indent: 100px;
+	margin-right: 80px;
+	text-align: center;
+	margin-top: 70px;
+}
+.test2{
+	font-style: semibold;
+	font-size: 18px;
+	text-indent: 300px;
+	margin-top: 0px;
+}
+.up{
+	height: 70px;
+	border: 1px solid black;
+	width: 94.75%;
+	background-color: #f9e42e;
+}
+#sign{
+	height: 50px;
+	text-align: center;
+	border: .5px solid black;
+	border-radius: 5px;
+	background-color: #033048;
+	width: 25%;
+	font-style: extra bold;
+	font-size: 12px;
+	line-height: 50px;
+	margin-top: 0px;
+	margin-left: 300px;
```

Comment:
This wont work on all screen sizes, use the margin auto

```
margin-left: auto;
margin-right: auto;
```

This will center anything that has a specified width